keps/sig-etcd/4326-downgrade/kep.yaml

```diff
@@ -3,7 +3,7 @@ kep-number: 4326
 authors:
   - "@serathius"
 owning-sig: sig-etcd
-status: implementable
+status: provisional
```

Comment:
@serathius - FYI

pushing back to provisional as PRR was not filled in and reviewed [to pass improved validation]

Reply:
✅

Reply:
SG